crypto/rsa/rsa_mp.c

```diff
@@ -93,3 +93,14 @@ int rsa_multip_calc_product(RSA *rsa)
     BN_CTX_free(ctx);
     return rv;
 }
+
+int rsa_multip_cap(int bits)
+{
+    if (bits < 1024)
+        return 2;
+    else if (bits < 4096)
```

Comment:
side-note, not required:  I hate else after return.

Reply:
How about now?